client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -580,6 +581,62 @@ public void testFlush() {
         assertThat(request.getMethod(), equalTo(HttpPost.METHOD_NAME));
     }
 
+    public void testForceMerge() {
+        {
+            String[] indices = randomIndicesNames(0, 5);
+            ForceMergeRequest forceMergeRequest = new ForceMergeRequest(indices);
+            Map<String, String> expectedParams = new HashMap<>();
+            setRandomIndicesOptions(forceMergeRequest::indicesOptions, forceMergeRequest::indicesOptions, expectedParams);
+            if (randomBoolean()) {
+                forceMergeRequest.maxNumSegments(randomInt());
+            }
+            expectedParams.put("max_num_segments", Integer.toString(forceMergeRequest.maxNumSegments()));
+            if (randomBoolean()) {
+                forceMergeRequest.onlyExpungeDeletes(randomBoolean());
+            }
+            expectedParams.put("only_expunge_deletes", Boolean.toString(forceMergeRequest.onlyExpungeDeletes()));
+            if (randomBoolean()) {
+                forceMergeRequest.flush(randomBoolean());
+            }
+            expectedParams.put("flush", Boolean.toString(forceMergeRequest.flush()));
+
+            Request request = Request.forceMerge(forceMergeRequest);
+            StringJoiner endpoint = new StringJoiner("/", "/", "");
+            if (indices.length > 0) {
+                endpoint.add(String.join(",", indices));
+            }
+            endpoint.add("_forcemerge");
+            assertThat(request.getEndpoint(), equalTo(endpoint.toString()));
+            assertThat(request.getParameters(), equalTo(expectedParams));
+            assertThat(request.getEntity(), nullValue());
+            assertThat(request.getMethod(), equalTo(HttpPost.METHOD_NAME));
+        }
+        {
+            ForceMergeRequest forceMergeRequestAll = new ForceMergeRequest();
+            Map<String, String> expectedParams = new HashMap<>();
```

Comment:
This test case seems the same as the one above L586-L612 when the `randomIndicesNames(0, 5)` returned an empty array ( in `BroadcastRequest.java` : `protected String[] indices = Strings.EMPTY_ARRAY` ).